hadoop-project/pom.xml

```diff
@@ -85,7 +85,7 @@
     <re2j.version>1.1</re2j.version>
 
     <!--Protobuf version for backward compatibility-->
-    <protobuf.version>2.5.0</protobuf.version>
+    <protobuf.version>3.7.1</protobuf.version>
```

Comment:
This could be unacceptable since it is pinned to 2.5.0 for compatibility by [HADOOP-16596](https://issues.apache.org/jira/browse/HADOOP-16596)(#1635). [HDFS-15790](https://issues.apache.org/jira/browse/HDFS-15790)(#2767) too is related.

Reply:
Thanks @iwasakims for the review. i have reverted the protobuf version to 2.5.0